guides/source/active_storage_overview.md

```diff
@@ -198,6 +200,27 @@ google:
   bucket: ""
 ```
 
+Optionally provide a Hash of upload options:
+
+```yaml
+google:
+  service: GCS
+  credentials: <%= Rails.root.join("path/to/keyfile.json") %>
+  project: ""
+  bucket: ""
+  upload:
+    acl: "" # will be set to `public_read` on public buckets
+    cache_control: ""
+    content_encoding: ""
+    content_language: ""
+    crc32c: ""
+    encryption_key: ""
+    kms_key: ""
+    metadata:
+      key: ""
+    storage_class: ""
```

Comment:
The `crc32c` option doesn’t make much sense to set at the service level.

Anyway, it’d probably be best not to have to keep this list up to date. Let’s provide one or two example options like we do for S3—`storage_class` seems like a good one—and link off to the GCS SDK docs for the full list.